vtex/faststore
# Feat: Next 13 - Global Section - `RegionBar` and `RegionModal` Section - SFS-1017  SFS-1013

## What's the purpose of this pull request?

This PR aims to migrate the `RegionBar` and `RegionModal`  from the pages router to the app router (next 13). 


## How to test it?

- Run the local server (`yarn dev` in core package);
- Make sure that all the current pages are working as expected (`/`, `/office` and some `/<slug>/p`);
- Make sure `/fs-next-update` is rendering with thew `RegionBar` section on the top of the page and without errors.
<img width="508" alt="Screenshot 2024-06-13 at 18 46 48" src="https://github.com/vtex/faststore/assets/11325562/9c0e0fb5-ab15-4348-a194-70d09602c7b2">
<img width="258" alt="Screenshot 2024-06-13 at 18 54 20" src="https://github.com/vtex/faststore/assets/11325562/8520c94b-3454-4e6f-aa84-8a8cd1ff3155">

<img width="508" alt="Screenshot 2024-06-13 at 18 47 05" src="https://github.com/vtex/faststore/assets/11325562/655ed175-9491-48ef-8471-4a8852b9412e">
